app.rb

```diff
@@ -39,8 +50,9 @@ def add_a_music_album
     raise StandardError, 'not implemented'
   end
 
-  def add_a_movie
-    raise StandardError, 'not implemented'
+  def add_a_movie(date, archived, silent, id: nil)
```

Comment:
Selecting this option gives the following error:
`add_a_movie': wrong number of arguments (given 0, expected 3) (ArgumentError)

Maybe in the main.rb, calling this function needs arguments.